### ISSUE TITLE
Source Control not pushing code to GitHub repo on WSL

### ISSUE THREAD

**Issue by Tetracyl** [microsoft/vscode#137145]
Issue Type: <b>Bug</b>

I'm using Vscode's source control feature in order to push my changes to my GitHub. The issue is that it never pushes any code and keeps loading forever. I am able to commit my changes but never able to push using the source control feature.

Using git commands to push my code in terminal works perfectly fine though. 

VS Code version: Code 1.62.2 (3a6960b964327f0e3882ce18fcebd07ed191b316, 2021-11-11T20:56:38.428Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No
Remote OS version: Linux x64 5.10.16.3-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.90GB (5.92GB free)|
|Process Argv|--crash-reporter-id cb01266e-7720-4367-8bc7-3c1efd25d28e|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.10.16.3-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz (8 x 1992)|
|Memory (System)|12.40GB (11.25GB free)|
|VM|0%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
vscode-custom-css|be5|6.0.1
vsc-material-theme|Equ|33.2.0
vsc-material-theme-icons|equ|1.2.2
discord-vscode|icr|5.7.0
Lua|key|0.0.9
Kotlin|mat|1.7.1
jupyter-keymap|ms-|1.0.0
remote-wsl|ms-|0.58.5
awesome-flutter-snippets|Nas|3.0.2
material-icon-theme|PKi|4.10.0
material-theme|zhu|3.13.2
vscode-eslint|dba|2.2.2
copilot|Git|1.7.3822
todo-tree|Gru|0.0.214

(9 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30384386
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis200cf:30386380
vsaa593cf:30376535
vssld246:30386377

```

</details>

**Comment by gjsjohnmurray** [comment]
Maybe #137063

**Comment by gjsjohnmurray** [comment]
/duplicate of #137183 which is scheduled to be fixed in upcoming recovery release (1.62.3)
